obspy/io/seiscomp/sc3ml.py

```diff
@@ -398,11 +401,11 @@ def _read_channel(inventory_root, cha_element, _ns):
     if sensor_element is not None:
         response_id = sensor_element.get("response")
         if response_id is not None:
-            resp_type = response_id.split("#")[0]
-            if resp_type == 'ResponsePAZ':
-                search = "responsePAZ[@publicID='" + response_id + "']"
-                response_element = inventory_root.find(_ns(search))
-            elif resp_type == 'ResponsePolynomial':
+            resp_type = 'ResponsePAZ'
+            search = "responsePAZ[@publicID='" + response_id + "']"
+            response_element = inventory_root.find(_ns(search))
+            if response_element is None:
+                resp_type == 'ResponsePolynomial'
```

Comment:
The sensor response attribute can only be either responzePAZ or responsePolynomial. ResponseFIR does not need to be searched because it can only be defined in a `digitalFilterChain` element inside a `datalogger` element and by then you already know the type.